en/ros/mavros_custom_messages.md

```diff
@@ -0,0 +1,348 @@
+# Sending Custom Message from MAVROS to PX4
+
+> **Warning** This article is not compatible with current development environment/stable versions. 
+  It has been tested against:
+  - **Ubuntu:** 14.04
```

Comment:
I have done it before for 16.04 and Melodic + 18.04. The steps are the same so this guide should generalize well

Reply:
Thanks @mrivi , @RicardoM17 is going to update so can remove this caveat (or more likely replace with latest versions when he does so)

Ricardo, please just add your changes directly into this branch/PR.